composeApp/build.gradle.kts

```diff
@@ -279,3 +273,15 @@ fun copyRecursive(
         }
     }
 }
+
+tasks.register("runDebug", Exec::class) {
```

Comment:
Why do we need this task?

Reply:
This enables us to set up custom gradle tasks with `composeApp:runDebug -Porganization=ooni` or `composeApp:runDebug -Porganization=dw` to run the various flavors without needing to edit files.

Reply:
I haven't tried any option that works